### DISCUSSION TITLE
How to properly make a face swap?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [lllyasviel/Fooocus#1239] [General]
How to properly make a face swap?

**Comment by Phrixus2023** [comment]
https://www.youtube.com/watch?v=wOBcNPFuB70&t=29s

**Comment by KitsonBroadhurst** [comment]
Made an English alternative:
https://youtu.be/nWvOzBG1k_k

**Comment by infinity0** [comment]
See #2644, Upscale works much better than Inpainting.
